usr/lib/cca_stdll/cca_specific.c

```diff
@@ -3524,11 +3624,13 @@ static CK_RV cca_get_min_card_level(STDLL_TokData_t *tokdata)
     }
 
     /* Update min card level in cca_private_data, needs write-lock. */
-    if (pthread_rwlock_wrlock(&cca_private->min_card_version_rwlock) != 0) {
-        TRACE_ERROR("CCA min_card_version RW-lock failed.\n");
+    rc = pthread_rwlock_wrlock(&cca_private->min_card_version_rwlock);
```

Comment:
this looks like an unrelated change. The rc variable is not really necessary, right? 
I guess this is a left over from a previous stage.
Maybe just revert it back to how it was originally.

Reply:
OK, is the errno really of much interest here? If it fails it fails..... there is not much you can do about that. 
We don't print it in the other millions of similar code places either..